function/random/random.js

```diff
@@ -4,49 +4,63 @@ module.exports = function(RED) {
     function RandomNode(n) {
         RED.nodes.createNode(this,n);
 
-        this.low  = Number(n.low) || ""; 
-        this.high = Number(n.high) || "";
+        this.status({}); // blank status on a deploy
+
         this.inte = n.inte || false;
         this.property = n.property||"payload";
         var node = this;
-        
-      
+
+
         this.on("input", function(msg) {
+ 
+           if (n.low !== "") {this.low  = parseFloat(n.low)}
+              else {this.low = n.low}; 
+           if (n.high !== "") {this.high  = parseFloat(n.high)}
+              else {this.high = n.high}; 
+      
+           if (node.low === "") {
+               node.low = 1;
+               if ('from' in msg) {node.low = msg.from} 
+           }
+                
+           if (node.high === "") {
+               node.high = 10; 
+               if ('to' in msg) {node.high = msg.to} 
+           }
+
+           // if returning an interger, do a parseInt() on the low and high values
+           if ( (node.inte == "true") || (node.inte === true) ) {
```

Comment:
not sure why we need to force the limits to be integers. Just because we set the output to be integers, the limits could still be floats. What is wrong with asking for an integer between 1.2 and 8.7 ?